src/OSPSuite.Core/Comparison/ExpressionProfileDiffBuilder.cs

```diff
@@ -0,0 +1,30 @@
+﻿using OSPSuite.Core.Domain.Builder;
+
+namespace OSPSuite.Core.Comparison
+{
+   public class ExpressionProfileBuildingBlockDiffBuilder : BuildingBlockDiffBuilder<ExpressionProfileBuildingBlock, ExpressionParameter>
+   {
+      public ExpressionProfileBuildingBlockDiffBuilder(ObjectBaseDiffBuilder objectBaseDiffBuilder, EnumerableComparer enumerableComparer) : base(objectBaseDiffBuilder, enumerableComparer)
+      {
+
+      }
+
+      public override void Compare(IComparison<ExpressionProfileBuildingBlock> comparison)
+      {
+         base.Compare(comparison);
+         CompareValues(x => x.PKSimVersion, x => x.PKSimVersion, comparison);
+         CompareStringValues(x => x.MoleculeName, x => x.MoleculeName, comparison);
+         CompareStringValues(x => x.Species, x => x.Species, comparison);
+         CompareStringValues(x => x.Category, x => x.Category, comparison);
+         CompareStringValues(x => x.Name, x => x.Name, comparison);
+         CompareValues(x => x.Type, x => x.Type, comparison, (x,y) => Equals(x.DisplayName, y.DisplayName), (x,y) => y.DisplayName);
```

Comment:
why the overlaod of equals here? Type is a static instance correct? They should be the same. Also if you override ToString in the type,e you do not need the DisplayName in the c comparison either

Reply:
@msevestre hehe possibly this is what you menat above? Removing it, again I will test the new outcome of the comparison

Reply:
Type is an 'ExpressionType' That means Enzyme, Transporter, BindingPartner

Reply:
We need that comparison put back in if you took it out, but if you want, rename the property `ExpressionType` instead of `Type` to avoid confusion.

Reply:
Ok, after thinking about it more, I get what you are saying. This `Type` is just doing a reference compare anyway, so why override, that was because I needed the DisplayName in the diff. Your second sentence is override ToString(), which solves that. So, indeed, we don't need this line back in.